recipes/parallel-preprocessor/meta.yaml

```
@@ -0,0 +1,80 @@
+# manually update or generated from template
```

It's better to use a variable for the version to make sure the URL stays in sync:
```suggestion
{% version = "0.3" %}

```